docs/docs/cmd/viva/engage/engage-role-list.mdx

```diff
@@ -0,0 +1,102 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# viva engage role list
+
+Lists all Viva Engage roles
+
+## Usage
+
+```sh
+m365 viva engage role list [options]
+```
+
+## Options
+
+<Global />
+
+## Remarks
+
+:::warning
+
+This command is based on an API that is currently in preview and is subject to change once the API reaches general availability.
+
+:::
+
+## Permissions
+
+<Tabs>
+  <TabItem value="Delegated">
+
+  | Resource        | Permissions                                           |
+  |-----------------|-------------------------------------------------------|
+  | Microsoft Graph | EngagementRole.Read.All, EngagementRole.ReadWrite.All |
+
+  </TabItem>
+  <TabItem value="Application">
+
+  | Resource        | Permissions                                           |
+  |-----------------|-------------------------------------------------------|
+  | Microsoft Graph | EngagementRole.Read.All, EngagementRole.ReadWrite.All |
```

Comment:
We only show the minimal permissions needed for the command. So in this case, we can remove the ReadWrite scope.